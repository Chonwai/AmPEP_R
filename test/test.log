Loading required package: seqinr
Loading required package: caret
Loading required package: lattice
Loading required package: ggplot2

Attaching package: ‘caret’

The following object is masked from ‘package:seqinr’:

    dotPlot

Loading required package: protr
Loading required package: MLmetrics

Attaching package: ‘MLmetrics’

The following objects are masked from ‘package:caret’:

    MAE, RMSE

The following object is masked from ‘package:base’:

    Recall

Loading required package: randomForest
randomForest 4.6-12
Type rfNews() to see new features/changes/bug fixes.

Attaching package: ‘randomForest’

The following object is masked from ‘package:ggplot2’:

    margin

Loading required package: gplots

Attaching package: ‘gplots’

The following object is masked from ‘package:stats’:

    lowess

Loading required package: methods
Type 'citation("pROC")' for a citation.

Attaching package: ‘pROC’

The following objects are masked from ‘package:stats’:

    cov, smooth, var


Attaching package: ‘psych’

The following object is masked from ‘package:randomForest’:

    outlier

The following objects are masked from ‘package:ggplot2’:

    %+%, alpha

Error in extractCTDD(seqs[i]) : x has unrecognized amino acid type
Calls: constructDescMatrix -> extractCTDD
Execution halted
