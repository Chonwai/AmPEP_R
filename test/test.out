AC_1 1 0.970000
AC_2 1 0.720000
